did_resolver_sov/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 
 [dependencies]
 did_resolver = { path = "../did_resolver" }
-aries_vcx_core = { path = "../aries_vcx_core", features = ["modular_libs"] }
+aries_vcx_core = { path = "../aries_vcx_core", features = ["credx"] }
```

Comment:
nit; resolve if you want: i think this could be `default-features = false` instead of `credx` dependency

Reply:
nvm i misread this as `aries-vcx`

Reply:
or just no features, since `aries_vcx_core` has no default features

Reply:
On it.

Reply:
Removed features.